src/sentry/rules/actions/base.py

```diff
@@ -98,3 +99,32 @@ def build_description(self, event, installation):
         return installation.get_group_description(event.group, event) + self.generate_footer(
             rule_url
         )
+
+    def has_linked_issue(self, event, integration):
+        linked_issues = GroupLink.objects.filter(
+            project_id=self.project.id,
+            group_id=event.group.id,
+            linked_type=GroupLink.LinkedType.issue,
+        ).values("linked_id")
+
+        integration_ids = [
```

Comment:
I think you can do: 
```
external_issue_ids = [l["linked_id"] for l in linked_issues]
return ExternalIssue.objects.filter(
	id__in=external_issue_ids,
	integration_id=integration.id,
).exists()
```
instead.

Reply:
Actually I'm looking at `.values_list` and it looks like you can pass it `flat=True` and it'll automatically give you a `list` of ids.